src/go/wsl-helper/pkg/wsl-utils/version.go

```diff
@@ -0,0 +1,290 @@
+//go:build windows
+// +build windows
+
+package wslutils
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"unsafe"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/windows"
+)
+
+// WSLInfo describes the current (online) WSL installation.
+type WSLInfo struct {
+	Installed bool           `json:"installed"`  // Whether WSL is considered to be installed.
+	Inbox     bool           `json:"inbox"`      // Whether WSL was shipped in-box or from the MS Store/MSIX
+	HasKernel bool           `json:"has_kernel"` // Whether WSL has a kernel installed
+	Version   PackageVersion `json:"version"`    // Installed WSL version (only for store version)
+}
+
+const (
+	kPackageFamily            = "MicrosoftCorporationII.WindowsSubsystemForLinux_8wekyb3d8bbwe" // spellcheck-ignore-line
```

Comment:
Needs a comment explaining where these two constants came from

Reply:
It is probably somebody's password that they accidentally pasted into a source file, and now it is too late to change the name because it would break backwards compatibility! 😛